issuer-list/src/main/java/com/adyen/checkout/issuerlist/IssuerListSpinnerView.kt

```diff
@@ -80,20 +92,26 @@ abstract class IssuerListSpinnerView<
     }
 
     override fun onItemSelected(parent: AdapterView<*>, view: View, position: Int, id: Long) {
-        d(TAG, "onItemSelected - " + issuersAdapter.getItem(position).name)
+        Logger.d(TAG, "onItemSelected - " + issuersAdapter.getItem(position).name)
         component.inputData.selectedIssuer = issuersAdapter.getItem(position)
         component.notifyInputDataChanged()
     }
 
     override fun setEnabled(enabled: Boolean) {
         super.setEnabled(enabled)
-        issuersSpinner.isEnabled = enabled
+        autoCompleteTextViewIssuers.isEnabled = enabled
+        textInputLayoutIssuers.isEnabled = enabled
     }
 
     override fun onNothingSelected(parent: AdapterView<*>) {
         // nothing changed
     }
 
+    private fun setupTermsAndConditionsHyperLink() {
+        termsAndConditionsTextView?.movementMethod = LinkMovementMethod.getInstance()
+        termsAndConditionsTextView?.setLinkTextColor(Color.BLUE)
```

Comment:
are we sure about this blue? shouldn't it be a custom blue?

Reply:
good catch I am not really sure.